www-client/luakit/luakit-2.3.1.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-1 luajit )
```

Comment:
Any chance to bump the lua compat above 5.1?

Reply:
I will have to look into upstream code for that. Good chance that it's possible.